doc/py_tutorials/py_gui/py_video_display/py_video_display.markdown

```diff
@@ -55,9 +55,9 @@ video) and full details can be seen here: cv::VideoCapture::get() .
 Some of these values can be modified using **cap.set(propId, value)**. Value is the new value you
 want.
 
-For example, I can check the frame width and height by cap.get(3) and cap.get(4). It gives me
-640x480 by default. But I want to modify it to 320x240. Just use ret = cap.set(3,320) and
-ret = cap.set(4,240).
+For example, I can check the frame width and height by `cap.get(3)` and `cap.get(4)`. It gives me
```

Comment:
while at it you could replace 3 and 4 by `cv2.CAP_PROP_FRAME_WIDTH` and `cv2.CAP_PROP_FRAME_HEIGHT`.